composer.json

```diff
@@ -30,6 +31,13 @@
             "SymfonyCasts\\Bundle\\ResetPassword\\Tests\\": "tests/"
         }
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "name": "symfonycasts/internal-test-helpers",
+            "url": "https://github.com/symfonycasts/internal-test-helpers"
+        }
+    ],
```

Comment:
We need the `repository` because `internal-test-helpers` doesn't exist on Packagist. We _could_ add it there, but I feel like that would open up a new can of worms with random's attempting to use it. Thoughts @bocharsky-bw

Reply:
It's an internal tool, and it works this way, so I'm fine with it and don't see any harm, that's for testing anyway 👍